examples/chemistry/excited_states.ipynb

```diff
@@ -1,1384 +1,1768 @@
+        "\n",
+        "![BODIPY](https://github.com/sandbox-quantum/Tangelo-Examples/blob/main/examples/img/bodipy_absorption.png?raw=1)\n",
+        "\n",
+        "As there are a very large number of compounds to be considered, predicting absorption/emission UV-visible spectra would be a valuable asset to the scientific community.\n",
+        "\n",
+        "To achieve complete understanding of light interaction with a molecule, the quantum chemistry community has worked on several algorithms. In general, one must compute the relevant molecular electronic structures for the prediction of UV light absorption/emission. This notebook shows how Tangelo enables excited states calculations by implementing a few existing quantum algorithms. These are broadly grouped into variational optimization algorithms and algorithms that rely on Hamiltonian simulation. Along the way, we keep track of the quantum computational resources required by each of these approaches, and summarize this information at the end of the notebook. The use case here is Li $_2$ for expediency but many of these quantum algorithms can, in principle, be extended to much larger systems such as the BODIPY molecule above.\n",
+        "\n",
+        "It is worth noting that even with all the computed excited states, non-trivial effects can happen (solvation effect, geometry change, etc.) in which all modify the shape of a spectrum. In this notebook, we do not discuss how these effects are accounted for, but the calculations presented here are the necessary first steps towards computing excited states."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "iujthgTEPHjH"
+      },
+      "source": [
+        "## Installation & Background\n",
+        "In order to successfully run this notebook, you need to install Tangelo. It is also important to be somewhat familiar with the variational quantum eigensolver (VQE). Information about VQE can be found in our [VQE with Tangelo](../variational_methods/vqe.ipynb) notebook. Information about each algorithm can be found by following the references linked when each method is introduced. The cell below installs Tangelo in your environment, if it has not been done already."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install --prefer-binary pyscf==2.3.0"
```

Comment:
Could we please add this cell after the one that installs Tangelo, preceded by a sentence that says 

"We'll also need to install `pycsf`, to support basic quantum chemistry features :"

Reply:
Of course! I'm creating another pull request. If you want me to change anything else please let me know!